src/main/scala/io/iohk/ethereum/db/storage/pruning/Basic.scala

```diff
@@ -0,0 +1,12 @@
+package io.iohk.ethereum.db.storage.pruning
+
+import io.iohk.ethereum.db.storage.{NodeStorage, ReferenceCountNodeStorage}
+import io.iohk.ethereum.utils.Logger
+
+case class Basic(history: Int) extends PruningMode with RangePrune with Logger {
```

Comment:
just a random thought, do we need those pruning modes. I mean do we need logic in them? Can't it be handled just as a part of `ReferenceCountNodeStorage`?